File: benchmark/python/sum_of_squares.py
import time

sum = 0
start = time.time() * 1000
for i in range(1, 1000001):
    sum = sum + i * i
end = time.time() * 1000
print(f"Sum: {sum}")
print(f"Time: {end - start:.0f}ms")

File: benchmark/php/sum_of_squares.php
<?php
$sum = 0;
$start = hrtime(true);
for ($i = 1; $i <= 1000000; $i++) { $sum += $i * $i; }
$end = hrtime(true);
$ms = ($end - $start) / 1e6;
echo "Sum: " . $sum . "
";
echo "Time: " . round($ms) . "ms
";
